bigquery_etl/cli/query.py

```diff
@@ -759,7 +747,20 @@ def schema():
     default="moz-fx-data-shared-prod",
     callback=is_valid_project,
 )
-def update(name, sql_dir, project_id):
+@click.option(
+    "--update-downstream",
+    "--update_downstream",
+    help="Update downstream dependencies. GCP authentication required.",
+    default=False,
+    is_flag=True,
+)
+@click.option(
+    "--tmp-dataset",
+    "--tmp_dataset",
+    help="GCP datasets for creating updated tables temporarily.",
+    default="analysis",
```

Comment:
For updating downstream dependencies, temporary tables with updated schemas will be created in GCP. I tried creating them in the `tmp` dataset, however the dryrun account doesn't have permissions to query this dataset. Not sure if there was a specific reason?

Reply:
I can't think of a reason. It's likely an oversight. cc @whd